### DISCUSSION TITLE
Annoying Horn

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#136] [Ideas]
Hey ! :) Would you be able to create a command that spawn in the horn and it would make it spam the sound that it mades ?

**Comment by Gloktober** [comment]
the ship horn *

**Comment by winstxnhdw** [comment]
Sure, but how should it work? Duration-based? One-time?

**Reply by Gloktober** [reply]
yea I think /horn duration would be perfect !

**Reply by winstxnhdw** [reply]
Done.

**Reply by Gloktober** [reply]
thank you ! did you make it so the horn appears if it's not already bought ?

**Reply by winstxnhdw** [reply]
yeap

**Reply by Gloktober** [reply]
crazy work ! thankyou

**Reply by Gloktober** [reply]
it only spawns the thing but doesn't make any sound

**Reply by winstxnhdw** [reply]
I don't think you can hear it. It's just for other players.

**Reply by Gloktober** [reply]
ohhh i see !

**Reply by winstxnhdw** [reply]
I just tried it, I can hear it though.

**Reply by Gloktober** [reply]
maybe it's a host only option ?

**Reply by winstxnhdw** [reply]
No, I just tried it in a run.

**Reply by Gloktober** [reply]
you gotta hold it ?

**Reply by winstxnhdw** [reply]
Nope

**Reply by Gloktober** [reply]
weird , works for me when i host but not when i join a lobby

**Reply by winstxnhdw** [reply]
Hmm, maybe it's overloading your network. I have just pushed a change such that it only pulls the cord once.

**Reply by Gloktober** [reply]
works fine now !
